__tests__/topics.test.js

```diff
@@ -210,6 +210,46 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+
+
+describe('/api/users', () => {
+  describe('GET /api/users', () => {
+    test('200: status code and contain the expected data type and fields', async () => {
+      const res = await request(app).get('/api/users');
+      expect(res.statusCode).toBe(200);
+      expect(Array.isArray(res.body)).toBe(true);
+      res.body.forEach(user => {
+        expect(typeof user.username).toBe('string');
+        expect(typeof user.name).toBe('string');
+        expect(typeof user.avatar_url).toBe('string');
+      })
+    });
+    test('Handles invalid URL or route', async () => {
+      const res = await request(app).get('/api/invalidroute');
+      expect(res.statusCode).toBe(404);
+    });
```

Comment:
If you already have this test somewhere else you don't need it again - this can also be in it's own describe block as it's not specific to the api/users endpoint, it is just an endpoint that does not match any you have defined in your app.js